src/algebraic_geometry/is_open_comap_C.lean

```diff
@@ -0,0 +1,67 @@
+/-
+Copyright (c) 2021 Damiano Testa. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Damiano Testa
+-/
+import algebraic_geometry.prime_spectrum
+import ring_theory.polynomial.basic
+/-!
+The morphism `Spec R[x] --> Spec R` induced by the natural inclusion `R --> R[x]` is an open map.
+-/
+
+open ideal polynomial prime_spectrum set
+
+namespace algebraic_geometry
+
+namespace polynomial
+
+variables {R : Type*} [comm_ring R]
+
+/-- Given a polynomial `f ∈ R[x]`, `image_of_Df` is the subset of `Spec R` where at least one
+of the coefficients of `f` does not vanish.  We will prove that `image_of_Df` is the image of
```

Comment:
Instead of `We will prove`, please mention explicitly the name of the lemma where you prove this.